src/FirePokemon.java

```diff
@@ -0,0 +1,69 @@
+public class FirePokemon extends Pokemon{
+
+    private String lightningColor;
```

Comment:
Deze variabele komt in elke subklasse voor, dan is dit niet het meest ideale niveau om deze te instantieren. Dan zou het logischer zijn als deze in de super klasse staat